heliopy/data/util.py

```diff
@@ -152,11 +153,10 @@ def processing_func(file, **processing_kwargs)
     data = timefilter(data, starttime, endtime)
     if extension == '.cdf':
         cdf = _load_local(raw_file)
-        if type(units) is dict:
-            units = cdf_units(cdf, keys=units)
-        else:
-            units = cdf_units(cdf)
-        return units_attach(data, units)
+        units_cdf = cdf_units(cdf, keys=keys)
+        if units is not None:
+            units_cdf.update(units)
```

Comment:
👍 nice, I like this approach. Can you just add a quick comment above saying what this if statement is doing?

Reply:
When we do this, it will add the new keys and overwrite any existing keys with the new units value we defined in the OrderedDict! I will add a comment above